### PR TITLE
Created a cardano chart

### PR DESCRIPTION
**Pull request by andyfiorini**

### CODE REVIEW

File: cardano/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    {{- end }}
+    {{- with .Values.tolerations }}
+      tolerations:
+{{ toYaml . | indent 8 }}
+    {{- end }}
+      volumes:
+      - name: data
+      {{- if .Values.persistence.enabled }}
+        persistentVolumeClaim:
+          claimName: cardano-data #{{ .Values.persistence.existingClaim | default (include "stellar-node.fullname" .) }}
```
Comment (valo) [inline cardano/templates/deployment.yaml]: Please name the name of the PVC configurable. Also I don't see the definition of the PersistenceVolumeClaim. Shouldn't it be included?

**Review by valo — CHANGES_REQUESTED** [review]

File: cardano/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    {{- end }}
+    {{- with .Values.tolerations }}
+      tolerations:
+{{ toYaml . | indent 8 }}
+    {{- end }}
+      volumes:
+      - name: data
+      {{- if .Values.persistence.enabled }}
+        persistentVolumeClaim:
+          claimName: {{ .Values.persistence.existingClaim | default (include "stellar-node.fullname" .) }}
```
Comment (valo) [inline cardano/templates/deployment.yaml]: In the common case we don’t have an existing PVC to put here. I think there should be a PVC definition as part of this chart. See 
https://github.com/santiment/helm-charts/blob/master/stellar-node/templates/pvc.yaml

**Comment by andyfiorini** [comment]
Ok, I will add the pvc.yaml in another pr.  Is that ok?  

I don't know how to add a new file to this pr now.

Regards.

**Comment by valo** [comment]
It should be possible to just push to your forked repo and the PR will update.

To speed up things I'll merge this one now

**Review by valo — APPROVED** [review]

**Pull request merged**